util/coverage/coverage.js

```diff
@@ -207,6 +207,13 @@ const miscStrings = {
   oopsy: {
     ...zoneGridHeaders.oopsy,
   },
+  // Description about release and latest version differences.
+  description: {
+    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date.',
```

Comment:
@quisquous 
```suggestion
    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date. Although this is mainly for developers previewing and testing, which means there might be few bugs. See also <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls"here</a> for more detals.',
```

I think if mentioned gh-pages, it should be nice to notice that the gh-pages version is not a stable one.

Reply:
I don't think it's mainly for developers, for what it's worth.  I think a number of users also use it to get changes quicker than releases happen, but probably not a huge number.

What about removing the added sentences and linking `quisquous.github.io version` to https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls?

Reply:
@quisquous 
```suggestion
    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">quisquous.github.io version</a> as the url for your overlays, this list will be up to date',
```

You mean like this?